capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.servlets;
+
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.step.clients.RestaurantClient;
+import com.google.step.data.Restaurant;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for sending/getting a restaurant to/from Datastore. */
+@WebServlet("/restaurant")
+public class RestaurantServlet extends HttpServlet {
+  private RestaurantClient restaurantClient = new RestaurantClient();
+
+  /**
+   * Will put the information gathered from the signup page and add the
```

Comment:
Nit: Consider rewording and remove the space before the period.

Reply:
Same here as the other one. I'll alter this in the other pull request.